.gitignore

```diff
@@ -3,3 +3,5 @@ node_modules
 .history
 plugin.js
 plugin.js.map
+browser.js
+browser.js.map
```

Comment:
Note to self: check how big the `.js.map` of the browser build is, and if it's big, then decide if it's worth it to include it (I don't think so)

Reply:
Sure, could leave it out.